package.json

```diff
@@ -41,7 +41,9 @@
     "ts-jest": "^25.2.0",
     "typescript": "^3.8.2"
   },
-  "resolutions": {},
+  "resolutions": {
```

Comment:
described in https://github.com/system-ui/theme-ui/pull/757#discussion_r391886922